lib/sedna/algorithms/multi_task_learning/multi_task_learning.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import joblib
+
+from .task_jobs.artifact import Model, Task, TaskGroup
+from sedna.datasources import BaseDataSource
+from sedna.backend import set_backend
+from sedna.common.log import LOGGER
+from sedna.common.config import Context
+from sedna.common.file_ops import FileOps
+from sedna.common.class_factory import ClassFactory, ClassType
+
+__all__ = ('MulTaskLearning',)
+
+
+class MulTaskLearning:
```

Comment:
without any docstring/log/comment, hard to understand this MulTaskLearning

Reply:
fix